easybuild/easyconfigs/h/HDFView/HDFView-2.14-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Binary'
+
+name = 'HDFView'
+version = '2.14'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://support.hdfgroup.org/products/java/hdfview/'
+description = "HDFView is a visual tool for browsing and editing HDF4 and HDF5 files."
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://support.hdfgroup.org/ftp/HDF5/hdf-java/current/bin/']
+sources = ['%(name)s-%(version)s-centos7_64.tar.gz']
```

Comment:
this package requires glibc 2.14, otherwise it works but doesn't support hdf5; should use centos6 package in systems with glibc 2.12

Reply:
So, should we include `centos7` in the `versionsuffix`?

Reply:
I saw some (old) CUDA easyconfigs where the `OS_NAME` and `OS_VERSION` is used to select the package, not sure if this is a good practice

otherwise yes, include `centos7` in the versionsuffix (and add an another easyconfig for `centos6`)?